src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2393,8 +2393,7 @@ def forward(
         mask_labels (`List[torch.Tensor]`, *optional*):
             List of mask labels of shape `(num_labels, height, width)` to be fed to a model
         class_labels (`List[torch.LongTensor]`, *optional*):
-            list of target class labels of shape `(num_labels, height, width)` to be fed to a model. They identify the
-            labels of `mask_labels`, e.g. the label of `mask_labels[i][j]` if `class_labels[i][j]`.
+            List of class labels of shape `(num_labels)` to be fed to a model.
```

Comment:
Should it just be `labels` similar to https://github.com/huggingface/transformers/blob/3b1be043cd2e754dec019547e36a0b97ccbfdcd7/src/transformers/models/mask2former/modeling_mask2former.py#L750

Reply:
@stevhliu

Reply:
Sorry I wasn't clear before! My question is should it be `num_labels` or `labels`? If it's the latter, then it needs to be changed :)

```suggestion
            List of class labels of shape `(labels)` to be fed to a model.
```

Reply:
@stevhliu It should be `labels`. Committed the suggestion.